### ISSUE TITLE
Add spinUp to momentum equations

### ISSUE THREAD

**Issue by srobertp** [erdc-cm/proteus-ps#29]
The requirements on our initial approximation of pressure and velocity are as follows:

Define the Stokes projection of (u(t),p(t)) to be the pair (S_h u(t),  S_h p(t)) in X_h x M_h that satisfies the projection of the non time dependent parts of the momentum equation.  Mainly,

< \mu grad S_h u(t), grad v_h >  + < grad S_h p(t), v_h >  =  < \mu grad u(t),  grad v_h> - <p(t), div v_h > for all v_h in X_h
and 
< S_h u(t) , grad r_h > = 0    for all r_h in M_h

where X_h is the finite element space for velocity and M_h for pressure.

Then one of the following properties must hold
(1)   |  p_h^0  - S_h p(0) |_{L^2(\Omega)}  <=  c h
or
(2)  |  p_h^0 |_{H^1(\Omega)}  < c  and  < u_h^0 , grad r_h > = 0  for all r_h in M_h.

In other words,  our initial pressure must be close to the stokes projection of pressure, or the velocity must be discretely divergence free.

We need to establish that one of these properties holds.

**Comment by srobertp** [comment]
With the spinUp property, we can possibly get our initial solutions for the momentum equations to satisfy one of these properties.
